### ISSUE TITLE
json logging?

### ISSUE THREAD

**Issue by ccmcbeck** [zalando-incubator/kube-ingress-aws-controller#561]
Is there a way to configure log output to be JSON rather than text?  Something like skipper's `-access-log-json-enabled`

**Comment by szuecs** [comment]
@ccmcbeck no, but this is only a control loop that does not have access logs and should not log much.
